test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -1,16 +1,15 @@
 package seedu.addressbook.common;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import org.junit.Test;
 
 import java.util.Arrays;
-import java.util.List;
 
-import org.junit.Test;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 public class UtilsTest {
 
-
+    //this is a method of UtilsTest that tests the method Utils.elementsAreUnique
```

Comment:
Violates code quality guidelines.
If you intend to add JavaDocs, it should be a block comment (/** ...*... */). Also, there should be a space after "//". 

Generally, test methods do not require JavaDocs as it is named after the method that it is testing.